net/fwknop/files/fwknopd.init

```diff
@@ -8,6 +8,8 @@
 USE_PROCD=1
 START=95
 
+. /lib/functions/network.sh
```

Comment:
Please move this into ``start_service()`` (just ahead of ``generate_configuration``).  This will avoid (non-fatal) messages during image building -- with OpenWrt there shouldn't be global sourcing on absolute paths because the code that enables the script actually runs the the script on the host (which obviously causes . /lib/functions/network.sh to emit an error message on the host).

Reply:
The variables in network.sh will exist for anything that executes after the sourcing, in the initscript.

Reply:
Created Issue https://github.com/openwrt/packages/issues/7884 for this.